lib/primer/view_components/engine.rb

```diff
@@ -13,7 +13,9 @@ class Engine < ::Rails::Engine
       ]
 
       config.primer_view_components = ActiveSupport::OrderedOptions.new
+
       config.primer_view_components.force_functional_colors = true
+      config.primer_view_components.force_system_arguments = true
```

Comment:
Would it make sense to keep the default value based on the environment?

Maybe `config.primer_view_components.force_system_arguments = !Rails.env.production?`

Reply:
I'll leave it false by default and let users decide